README.md

```diff
@@ -44,17 +44,18 @@ To setup your Docker OPX image, use the script in the _opx-build/scripts_ folder
 Go to the root directory where you installed the OPX repositories and run the OPX Docker container:
 
     cd ~/dev/opx
-    ./opx-build/scripts/opx_run
+    opx-build/scripts/opx_run
 
 To build a single repository, go to the repository and build. For example, to build `opx-logging`:
 
-    opx-dev@077f7b30f8ef:/# cd /mnt/opx-logging
-    opx-dev@077f7b30f8ef:/mnt/opx-logging# git-buildpackage --git-dist=jessie --git-ignore-branch --git-pbuilder
+    opx-dev@077f7b30f8ef:/# cd /mnt
+    opx-dev@077f7b30f8ef:/mnt# opx-build/scripts/opx_build opx-logging
```

Comment:
shouldn't it be: opx-dev@077f7b30f8ef:/mnt# opx-build/scripts/opx_build_all opx-logging